### PR TITLE
tickets/PREOPS-1232

### PR DESCRIPTION
**Pull request by MelissaGraham**
Completed update of the Introduction to the Portal Aspect, which covers basic search functionality.

### CODE REVIEW

**Comment by MelissaGraham** [comment]
A placeholder for image queries has been added.

File: data-access-analysis-tools/portal-intro.rst [hunk 1]
```diff
@@ -148,75 +179,79 @@ The language is used by the `IVOA <https://ivoa.net>`_ to represent astronomy qu
...
+.. figure:: /_static/portal_ImageQueryDP02.png
+    :name: portal_ImageQueryDP02
+
+    An early version of the Portal's image query interface.  
+
+
+.. figure:: /_static/portal_ImageQueryCoverageDP02.png
+    :name: portal_ImageQueryCoverageDP02
+
+    An early version of the Portal's image query search results coverage map.
```
Comment (kadrlica) [inline data-access-analysis-tools/portal-intro.rst:255]: Not sure you want to add any more, but I think that this image is showing the outlines of a coadd patch (orange) and calexps (pink) that cover a queried point.

**Review by kadrlica — APPROVED** [review]
Looks good! I've added some minor comments. I was having some transitory trouble executing the queries, which may have been data-int development wrinkles or user error.

File: data-access-analysis-tools/portal-intro.rst [hunk 2]
```diff
@@ -105,34 +124,46 @@ The search will return data in columns ``ra``, ``dec``, ``mag_g``, and ``mag_i``
...
 Select other columns to use, change the symbol type and color, and so forth, and click "Apply".
 
-.. figure:: /_static/portal_results_xy_settings.png
-    :name: portal_results_xy_settings
+**Getting Magnitudes from Fluxes**:
+The flux data in the DP0.2 tables are given in units of nanojansky, but in the future apparent magnitudes will be available.
+In the meantime, users can still obtain magnitudes and, as in this example, plot a color-magnitude diagram of the objects
+selected according to our query.
+As shown below, the following conversion is required to plot the retrieved flux values as a color-magnitude diagram:  
+``-1.0857*log(r_calibFlux/3.631e12)+1.0857*log(i_calibFlux/3.631e12)`` in the "X" box,
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-access-analysis-tools/portal-intro.rst]: Yup, turns out we can. Updated everything.

File: data-access-analysis-tools/portal-intro.rst [hunk 3]
```diff
@@ -105,34 +124,46 @@ The search will return data in columns ``ra``, ``dec``, ``mag_g``, and ``mag_i``
...
+and ``-1.0857log(g_calibFlux/3.631e12)`` box for the "Y" box.
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-access-analysis-tools/portal-intro.rst]: This part has been changed anyway.

File: data-access-analysis-tools/portal-intro.rst [hunk 4]
```diff
@@ -28,75 +28,94 @@ Log in to the Portal Aspect by clicking on the "Portal" panel of the main landin
...
-The longitude and latitude columns will automatically update to be the correct column names for right ascension and declination for the selected table (usually ``ra`` and ``dec`` or ``coord_ra`` and ``coord_dec``).
+**4. Enter Constraints**: For "Spatial" constraints, the longitude and latitude columns do not yet automatically 
+update to be the correct column names for right ascension and declination for the selected table, and need to be 
+entered (for the DP0.2 Object table, they are "coord_ra" and "coord_dec").
 If a non-existent column name is entered the box will highlight red in indication of the error.
-Choose the desired shape type for a spatial search, ``Cone`` or ``Polygon``, and the appropriate instructions for the search terms will appear.
-Keeping the search area to a minimum will keep processing times short and returned subsets small and manageable.
+Choose the desired shape type for a spatial search, such as "Cone", and the appropriate instructions for the search terms will appear.
+Keeping the search area to a minimum will keep query times short and returned subsets small and manageable as you learn.
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-access-analysis-tools/portal-intro.rst]: adopted

File: data-access-analysis-tools/portal-intro.rst [hunk 5]
```diff
@@ -28,75 +28,94 @@ Log in to the Portal Aspect by clicking on the "Portal" panel of the main landin
...
+(see the :ref:`DP0-2-Data-Products-DPDD` for a reminder of what tables are available).
+On the right-hand side of the panel, select "dp02_dc2_catalogs.Object" from the drop-down menu.  
 The table view at lower-right will automatically update to match the selected table.
 
-**4. Enter Constraints**: Only "Spatial" constraints apply for DP0.2.
-The longitude and latitude columns will automatically update to be the correct column names for right ascension and declination for the selected table (usually ``ra`` and ``dec`` or ``coord_ra`` and ``coord_dec``).
+**4. Enter Constraints**: For "Spatial" constraints, the longitude and latitude columns do not yet automatically 
+update to be the correct column names for right ascension and declination for the selected table, and need to be 
+entered (for the DP0.2 Object table, they are "coord_ra" and "coord_dec").
 If a non-existent column name is entered the box will highlight red in indication of the error.
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-access-analysis-tools/portal-intro.rst:65]: added

File: data-access-analysis-tools/portal-intro.rst [hunk 6]
```diff
@@ -105,34 +124,46 @@ The search will return data in columns ``ra``, ``dec``, ``mag_g``, and ``mag_i``
...
-    :name: portal_results_xy_settings
+**Getting Magnitudes from Fluxes**:
+The flux data in the DP0.2 tables are given in units of nanojansky, but in the future apparent magnitudes will be available.
+In the meantime, users can still obtain magnitudes and, as in this example, plot a color-magnitude diagram of the objects
+selected according to our query.
+As shown below, the following conversion is required to plot the retrieved flux values as a color-magnitude diagram:  
+``-1.0857*log(r_calibFlux/3.631e12)+1.0857*log(i_calibFlux/3.631e12)`` in the "X" box,
+and ``-1.0857log(g_calibFlux/3.631e12)`` box for the "Y" box.  
+Here, the log function in the "expression" for the plot is actually a natural log, and so the 1.0857 factor is a ratio of 2.5 over ln(10).
+The 3.631 is the standard AB conversion factor from flux to magnitude.
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-access-analysis-tools/portal-intro.rst]: This has changed to just use log10.

File: data-access-analysis-tools/portal-intro.rst [hunk 7]
```diff
@@ -105,34 +124,46 @@ The search will return data in columns ``ra``, ``dec``, ``mag_g``, and ``mag_i``
...
-.. figure:: /_static/portal_results_xy_settings.png
-    :name: portal_results_xy_settings
+**Getting Magnitudes from Fluxes**:
+The flux data in the DP0.2 tables are given in units of nanojansky, but in the future apparent magnitudes will be available.
+In the meantime, users can still obtain magnitudes and, as in this example, plot a color-magnitude diagram of the objects
+selected according to our query.
+As shown below, the following conversion is required to plot the retrieved flux values as a color-magnitude diagram:  
+``-1.0857*log(r_calibFlux/3.631e12)+1.0857*log(i_calibFlux/3.631e12)`` in the "X" box,
+and ``-1.0857log(g_calibFlux/3.631e12)`` box for the "Y" box.  
+Here, the log function in the "expression" for the plot is actually a natural log, and so the 1.0857 factor is a ratio of 2.5 over ln(10).
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-access-analysis-tools/portal-intro.rst]: It's updated, yeah.

File: data-access-analysis-tools/portal-intro.rst [hunk 8]
```diff
@@ -148,75 +179,79 @@ The language is used by the `IVOA <https://ivoa.net>`_ to represent astronomy qu
...
+   SELECT src.ccdVisitId, src.extendedness, src.band,
+   scisql_nanojanskyToAbMag(src.psfFlux) AS psfAbMag,
+   cv.obsStartMJD, cv.seeing 
+   FROM dp02_dc2_catalogs.Source AS src
+   JOIN dp02_dc2_catalogs.CcdVisit AS cv
+   ON src.ccdVisitId = cv.ccdVisitId
+   WHERE CONTAINS(POINT('ICRS', coord_ra, coord_dec), CIRCLE('ICRS', 62.0, -37, 1)) = 1
+   AND src.band = 'i' AND src.extendedness = 0 AND src.psfFlux > 10000
+   AND cv.obsStartMJD > 60925 AND cv.obsStartMJD < 60955
+
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-access-analysis-tools/portal-intro.rst]: This little demo has been removed.

File: data-access-analysis-tools/portal-intro.rst (hunk 1 above)
Comment (MelissaGraham) [reply data-access-analysis-tools/portal-intro.rst]: Oh that was just a really early placeholder, I added much much more.

**Pull request merged**